log4j-core/src/test/java/org/apache/logging/log4j/core/LoggerDateTest.java

```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertTrue;
```

Comment:
static imports go last. We allow wildcard imports so this change isn't necessary.